app/src/main/java/org/fossasia/susi/ai/skills/groupwiseskills/GroupWiseSkillsFragment.kt

```diff
@@ -73,6 +73,14 @@ class GroupWiseSkillsFragment : Fragment(), IGroupWiseSkillsView, SwipeRefreshLa
         progressSkillWait.visibility = if (boolean) View.VISIBLE else View.GONE
     }
 
+    override fun showEmptySkillsListMessage() {
+        if (activity != null) {
```

Comment:
what if activity is not null?

Reply:
That's when this'll get executed

Reply:
@batbrain7 Please have a look at this and see if this can be merged.

Reply:
Actually I meant if it was null

Reply:
If activity is null, this would never be called.

Reply:
This would only be called if everything is fine, but a particular group has no skills yet i.e. no skills exist for that group yet.